recipes/rdp-tools/meta.yaml

```diff
@@ -0,0 +1,37 @@
+
+package:
+  name: rdp-tools
+  version: "2.0.2"
+
+source:
+  git_url: https://github.com/rdpstaff/RDPTools.git
+  git_rev: 2.0.2
+
+requirements:
+  build:
+    - ant
+
+build:
+  number: 0
+  script:
+#    - brew update              # [osx]
+#    - brew install ant         # [osx]
+
+    - git config --global url."https://".insteadOf git://
+    - git submodule init
+    - git submodule update
+
+    - make
+
+    - mkdir -p $PREFIX/bin
+    - mv *.jar $PREFIX/bin
+    - mv lib $PREFIX/bin
+
+test:
+  commands:
+    - java -Xmx1g -jar $PREFIX/bin/classifier.jar classify -c 0.5 -o usga_classified.txt -h soil_hier.txt $SRC_DIR/classifier/samplefiles/USGA_2_4_B_trimmed.fasta
```

Comment:
Unfortunately, RDP tools does not have this kind of wrapper. 😞  This is their example command: https://github.com/rdpstaff/classifier/blob/master/README#L84 

I'll see if $PREFIX works or if there is a suitable replacement.